### PR TITLE
Reviews model

### PR DESCRIPTION
**Pull request by CharlesOuverleaux**
Reviews belong to a user and a spot, they have a title (max length to avoid too long text) and a description (min length to avoid too short text) and a rating from 1 to 5.

Seeding is adding 3..6  reviews to each existing spot, with a random: user, title, description and rating

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -69,14 +70,7 @@ def assign_random_review(spot)
   assign_random_review(item)
```
Comment (hannesschaletzky) [inline db/seeds.rb:70]: Print reviews as well

Comment (hannesschaletzky) [inline db/seeds.rb:70]: `puts " added: #{item[:name]}, #{item.reviews.count} review/s"`

**Review by hannesschaletzky — COMMENTED** [review]
Print out reviews count

File: db/seeds.rb (hunk 1 above)
Comment (CharlesOuverleaux) [reply db/seeds.rb:69]: Added in the seed file!

**Pull request merged**